python/cudf/cudf/core/column/string.py

```diff
@@ -4201,6 +4201,8 @@ def subword_tokenize(
         ----------
         hash_file : str
             Path to hash file containing vocabulary of words with token-ids.
+            This can be created from the raw vocabulary
+            using the `cudf.utils.hash_vocab_utils.hash_vocab` function
```

Comment:
Need to double tick in docstrings for proper sphinx formatting:
```suggestion
            using the ``cudf.utils.hash_vocab_utils.hash_vocab`` function
```